core/src/main/java/jenkins/util/SetContextClassLoader.java

```diff
@@ -41,7 +41,7 @@
  * unclear, {@link #SetContextClassLoader(ClassLoader)} can be used as a fallback with {@link
  * PluginManager.UberClassLoader} as the argument, though this is not as safe since lookups could be
  * ambiguous in case two unrelated plugins both bundle the same library. In functional tests, {@code
- * RealJenkinsRule.Endpoint} can be used to reference a class loader that has access to the plugins
+ * RealJenkinsFixture.Endpoint} can be used to reference a class loader that has access to the plugins
  * defined in the test scenario.
```

Comment:
Similarly here, you would rather use `PluginManager.uberClassLoader` to access classes defined in plugins loaded by the test. `Endpoint.class.classLoader` would be something broader, including test classes which are _not_ in plugins, and even if you did actually want that, I would never have recommended using `Endpoint.class.classLoader` which is an implementation detail of the test harness; rather specify a class constant that you expect to be found by some library code relying on the CCL. So this Javadoc should be amended as well. And then these two edits related to `SetContextClassLoader` could be made now, not waiting for a JTH release.

Reply:
Addressed in https://github.com/jenkinsci/jenkins/pull/26020